ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -127,10 +125,14 @@ export const AccountListItem = ({
                   position="bottom"
                   wrapperClassName="multichain-account-list-item__tooltip"
                 >
-                  {identity.name}
+                  <a href="#" onClick={onClick}>
+                    {identity.name}
+                  </a>
```

Comment:
Can we use ButtonLink here instead of `a`?

Reply:
I'm curious about whether ButtonLink is good here too

Reply:
I think the intention here is to keep the UI looking like static text but allow for keyboard navigation and tabbing. I guess the difference to this and GitHub is that they're using it for navigation while here we are using an onClick to change the state of the app not necessarily the route